### PR TITLE
Feature/28 image store type

### PR DESCRIPTION
**Pull request by ckertam**

### CODE REVIEW

File: backend/frontend/src/pages/story/CreateStory.js [hunk 1]
```diff
@@ -32,14 +33,25 @@ function CreateStory() {
...
+      ],
+    },
+    imageCompress: {
+      quality: 0.7, // default
+      maxWidth: 1024, // default
+      maxHeight: 1024, // default
+      imageType: 'image/jpeg', // default
+      debug: false, // default
+    }
+};
```
Comment (sadikkuzu) [inline backend/frontend/src/pages/story/CreateStory.js:54]: @ayhncvdr hi,
Here are the image compression lines done by @ckertam

Comment (ayhncvdr) [reply backend/frontend/src/pages/story/CreateStory.js:54]: I am not sure if we should set max width and height, does this affect the aspect ratio of the images?

Comment (sadikkuzu) [reply backend/frontend/src/pages/story/CreateStory.js:54]: Well, actually, in @ckertam 's example he got 768x1024 resolution. This module keeps the ratio.

Comment (ckertam) [reply backend/frontend/src/pages/story/CreateStory.js:54]: It's resolved I guess. We decided to use this with @ayhncvdr2 and we will also do it in mobile part! 🚀

**Review by sadikkuzu — APPROVED** [review]
LGTM

**Pull request merged**